pkg/workflows/workflows.go

```diff
@@ -1,7 +1,6 @@
 package workflows
```

Comment:
My plan was to remove these old template handlers completely, check out my PR on the addons migration here: https://github.com/Azure/draft/pull/406/files

I added the custom logic into the cmdhelper package as a placeholder until we have better repo analysis

Reply:
Also make sure you remove the `workflow.go` template file

Reply:
done